commit 200d1743dcdacf1036384f746046e55d91ccd918
Author: Eskil Abrahamsen Blomfeldt <eskil.abrahamsen-blomfeldt@nokia.com>
Date:   Fri Sep 3 12:48:01 2010 +0200

    Fix QStaticText copy constructor to also copy text option property

    The text option property of QStaticText was not copied in the copy
    constructor, so when the text was detached, the property would be
    reset to the default.

    Task-number: QTBUG-13368
    Reviewed-by: Gunnar

--- a/src/gui/text/qstatictext.cpp
+++ b/src/gui/text/qstatictext.cpp
@@ -392,9 +392,9 @@ QStaticTextPrivate::QStaticTextPrivate()
 
 QStaticTextPrivate::QStaticTextPrivate(const QStaticTextPrivate &other)
     : text(other.text), font(other.font), textWidth(other.textWidth), matrix(other.matrix),
-      items(0), itemCount(0), glyphPool(0), positionPool(0), charPool(0), needsRelayout(true),
-      useBackendOptimizations(other.useBackendOptimizations), textFormat(other.textFormat),
-      untransformedCoordinates(other.untransformedCoordinates)
+      items(0), itemCount(0), glyphPool(0), positionPool(0), charPool(0), textOption(other.textOption),
+      needsRelayout(true), useBackendOptimizations(other.useBackendOptimizations),
+      textFormat(other.textFormat), untransformedCoordinates(other.untransformedCoordinates)
 {
 }
 
